source/reference/collection-level-filtering.txt

```diff
@@ -42,31 +116,54 @@ Configure a Filter
 
    .. step:: Create a Filter.
 
-      The ``includeNamespaces`` parameter specifies an optional filter
-      that you can pass to the :ref:`/start <c2c-api-start>` API.
+      The :ref:`c2c-api-start` API accepts two parameters that configure
+      optional filters: 
       
-      If you don't specify a filter, ``mongosync`` does a full cluster
+      - The ``includeNamespaces`` parameter takes an array of filters, which
+        are used to determines which databases and collections ``mongosync`` 
+        should include in the sync
+      - The ``excludeNamespaces`` parameter takes an array of filters, which
+        are used to determine which databases and collections ``mongosync``
+        should exclude from the sync.
+     
+      If you don't specify a filter, ``mongosync`` performs a full cluster
       sync.
 
-      The filter syntax is: 
-
-      .. include:: /includes/api/facts/includeNamespaces-syntax.rst
-
-      Create an entry in the ``includeNamespaces`` array for each
-      database that you identified in step 1. Use the ``"database"``
-      field to specify the database name.
-      
-      If you want to filter on collections within a database, add those
-      collections to the list in the ``"collections"`` field for that
-      database entry.
+      Create inclusion and/or exclusion filters to identify the databases and
+      collections you want sync.
+
+      For example, this inclusion filter would configure ``mongosync`` to only
+      sync collections whose names begin with ``accounts_`` from the ``sales`` 
+      database, except for the ``accounts_old`` collection:
+
+      .. code-block:: json
+
+         "includeNamespaces": [
+            {
+               "database": "sales",
+               "collectionRegex": {
+                  "pattern": "^accounts_.+$",
+                  "options": "ms"
```

Comment:
This is a nit, so I'm fine with whatever you prefer. But I think `ms` is a weird example for collection names which are unlikely to have newlines. I think maybe `i` is a better example. Doesn't really matter, but thought I'd mention it.

Reply:
I'll leave it for now to avoid reworking the surrounding text, but will keep it in mind.